Explorer/Assets/DCL/Minimap/SceneRestrictionsController.cs

```diff
@@ -71,7 +71,7 @@ private void ManageSceneRestrictions(SceneRestriction sceneRestriction)
 
             restrictionsRegistry[sceneRestriction.Type] = currentRestrictionCounter;
 
-            restrictionsGameObjects[sceneRestriction.Type].SetActive(currentRestrictionCounter > 0);
+            restrictionsGameObjects[sceneRestriction.Type]?.SetActive(currentRestrictionCounter > 0);
```

Comment:
It looks like `nullity` check is redundant - you always create an instance before putting it into a dictionary